stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.7.1
+version: 1.8.0
```

Comment:
@djsly I think I'd argued on your previous PR that you needed a _patch_ version bump (the "z" in "x.y.z"). You were just upgrading to a newer version of Traefik, which itself was only a patch release. That meant the net effect on the chart was bug fixes.

This case is different. A minor version bump (the "y" in "x.y.z") is called for because this change introduces a new feature _to the chart_ (even if this feature isn't new to the underlying software).